integration/test/util.py

```diff
@@ -109,6 +109,16 @@ def skip_unless_do_launch():
         return unittest.skip("Most tests in this suite require launch; do not set --skip-launch.")
     return lambda func: func
 
+def skip_unless_levelzero_power():
+    #try/catch read signal for levelzero
+    try:
+        power = geopm_test_launcher.geopmread("LEVELZERO::POWER board_accelerator 0")
```

Comment:
This will result in an error on systems without LEVELZERO support or without GPUs, and catches the error to indicate we should skip the test.  
We may want to find a way to suppress the original error print to avoid confusion

Reply:
The error message should be immediately followed by a message saying that a test will be skipped.  I think this should be clear for a user, and the error message should not be suppressed.